assets/stylesheets/objects/_page-section.scss

```diff
@@ -0,0 +1,4 @@
+.page-section {
+  @extend %site-width-container;
+  padding: ($default-spacing-unit * 2) 0;
```

Comment:
There is a mixin called `element-spacing` which helps keeps vertical spacing consistent across screen sizes. We're currently using the default unit for mobile screen sizes then `* 2` on larger screens.

It might need tweaking to support padding though as I can see this needs to be padding to include the background colour.